components/ListSurfspotsItems.js

```diff
@@ -1,19 +1,24 @@
 import styled from "styled-components";
 import Link from "next/link";
+import Image from "next/image";
+import arrow from "../public/images/arrow.png";
 
 styled;
```

Comment:
a lonely "styled" out in the wild. Can this be deleted?

Reply:
oh yes :D